### ISSUE TITLE
TypeError: __init__() takes 4 positional arguments but 5 were given

### ISSUE THREAD

**Issue by az0** [stephenhky/PyShortTextCategorization#71]
I installed shorttext==1.1.3 on Python 3.7.3  on Ubuntu, and I ran the follow script adapted from the tutorial.  However, it gives a traceback

```Python
import numpy as np
import shorttext
from urllib.request import urlopen
response=urlopen('http://norvig.com/big.txt')
data=response.read()
text=data.decode('utf-8')
chartovec_encoder = shorttext.generators.initSentenceToCharVecEncoder(text)
latent_dim=5
seq2seqer = shorttext.generators.CharBasedSeq2SeqGenerator(chartovec_encoder, latent_dim, 120)
```

Traceback
```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/tmp/st/lib/python3.7/site-packages/shorttext/generators/seq2seq/charbaseS2S.py", line 47, in __init__
    self.s2sgenerator = Seq2SeqWithKeras(self.nbelem, self.latent_dim)
  File "/tmp/st/lib/python3.7/site-packages/shorttext/generators/seq2seq/s2skeras.py", line 41, in __init__
    cio.CompactIOMachine(self, {'classifier': 'kerasseq2seq'}, 'kerasseq2seq', kerasseq2seq_suffices)
TypeError: __init__() takes 4 positional arguments but 5 were given
```

**Comment by stephenhky** [comment]
Thanks for pointing out. I made the bug fix to the master class.

I will deploy a new version later today. Thanks!

**Comment by stephenhky** [comment]
Bugs fixed, and release 1.1.4 is out.
